compose.yml

```diff
@@ -21,50 +23,56 @@ services:
       - "-prune=0"
       - "-dbcache=4000"
       - "-mempoolfullrbf=1"
-    volumes:
-      - data:/bitcoin
+
 
   addrindexrs:
-    image: counterparty/addrindexrs:v0.4.5
+    image: counterparty/addrindexrs:v0.4.6
     links:
       - bitcoind
     volumes:
+      - data:/root/.bitcoin
       - data:/data
     ports:
-      - "8432:8432"                           # mainnet
-      # - "18432:18432"                       # testnet
+      - "8432:8432"                                        # mainnet
+      - "18432:18432"                                      # testnet
     command:
-      - "--network=bitcoin"                   # mainnet
-      # - "--network=testnet"                 # testnet
+      - "--network=${COUNTERPARTY_NETWORK:-main}"          # export COUNTERPARTY_NETWORK=test for testnet
       - "--indexer-rpc-host=0.0.0.0"
       - "--daemon-rpc-host=bitcoind"
       - "--cookie=rpc:rpc"
-      - "--daemon-dir=/bitcoin/.bitcoin"
       - "-vvv"
-      - "--db-dir=/data/addrindexrs/db"
+      - "--db-dir=/data/"
     environment:
-      - ADDRINDEXRS_JSONRPC_IMPORT=true
+      - ADDRINDEXRS_JSONRPC_IMPORT=${ADDRINDEXRS_JSONRPC_IMPORT:-false}
+
 
   counterparty-core:
     image: counterparty/counterparty:v10.0.0-beta.1
     links:
       - bitcoind
       - addrindexrs
     volumes:
+      - data:/root/.bitcoin
       - data:/data
     ports:
-      - "4000:4000"                           # mainnet
-      # - "14000:14000"                       # testnet
+      - "4000:4000"                                        # mainnet
+      - "14000:14000"                                      # testnet
```

Comment:
is it possible to do this as an inline Bash conditional based on the value of `COUNTERPARTY_NETWORK`?!

Reply:
hum.. I don't think.. we don't have all the base funcitonnality here.. we could just have a default value and pass 1400 when testnet.. IMO is not a problem to expose a unused port here.. if really needed security should be done with a firewall on another level..